### PR TITLE
Use none platform (empty) for agentCluster clusterDeployment

### PR DESCRIPTION
**Pull request by eranco74**

### CODE REVIEW

**Comment by eranco74** [comment]
/hold
need to check the assisted-service log and verify who approved the node CSR

File: controllers/agentcluster_controller.go [hunk 1]
```diff
@@ -195,9 +194,6 @@ func (r *AgentClusterReconciler) createClusterDeploymentObject(agentCluster *cap
 		Spec: hivev1.ClusterDeploymentSpec{
 			Installed:   true,
 			ClusterName: controlPlane.ClusterName,
-			Platform: hivev1.Platform{
-				AgentBareMetal: &agent.BareMetalPlatform{},
-			},
```
Comment (osherdp) [inline controllers/agentcluster_controller.go:200]: Can we verify this new behavior with a unit-test?

**Review by osherdp — CHANGES_REQUESTED** [review]

**Comment by eranco74** [comment]
This won't do the trick (get the CSR approved by the agent controller), we should set the user managed networking to false in the ACI instead.
/close

**Pull request closed without merging**